plugin/storage/cassandra/options.go

```diff
@@ -48,6 +48,8 @@ const (
 
 	// common storage settings
 	suffixSpanStoreWriteCacheTTL = ".span-store-write-cache-ttl"
+	suffixWritesPerSecond        = ".writer-rate-limit.operations-per-second"
```

Comment:
I'm ok with this for now but I think these configs should be storage type agnostic (ie ES can also use the rate limiter but we shouldn't have to pollute ES storage configs with `.writer-rate-limit.operations-per-second` and `.writer-rate-limit.max-burst`)

Reply:
I wish the configs generally were less storage specific. I'm not sure if we share any config at the moment.